src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`True with 'isog'`, () => {
```

Comment:
Consider providing more descriptive test case names. For example, instead of `True with 'isog'`, you could use `returns true for an isogram word`.

Reply:
I agree, it's better. But Misha in the video said that it is convenient to use short descriptions to easily understand which test is failed. You must not expand the test window in VSCode to read long test descriptions.